.circleci/config.yml

```diff
@@ -13,7 +13,9 @@ jobs:
           key: v1-deps-{{ checksum "pom.xml" }}
           paths:
             - ~/.m2
-      - run: mvn verify coveralls:report -Dcoveralls.token=${COVERALLS_TOKEN}
+      - run: |
+          mvn verify
+          bash <(curl -s https://codecov.io/bash)
```

Comment:
ignore the aggregate report?

Perhaps also separate test run and coverage submission into separate circleci run steps to make it easier to see output of the separate commands in the circleci ui.

```
- run: mvn verify
- run: bash <(curl -s https://codecov.io/bash -f '!report/target/site/jacoco-aggregate/jacoco.xml')
```